### ISSUE TITLE
Sequences in entities do not use quarkus.hibernate-orm.database.default-schema

### ISSUE THREAD

**Issue by olegsmetanin** [quarkusio/quarkus#31926]
### Describe the bug

Sequences in entities do not use quarkus.hibernate-orm.database.default-schema and you have to define the schema name in the sequence. Maybe https://github.com/hibernate/hibernate-reactive/issues/1120 is not fixed.

### Expected behavior

Sequences in entities use quarkus.hibernate-orm.database.default-schema

### Actual behavior

Sequences in entities do not use quarkus.hibernate-orm.database.default-schema

### How to Reproduce?

Steps to reproduce:
1) Clone this repository and cd to hibernate-reactive-quickstart
```
git clone https://github.com/quarkusio/quarkus-quickstarts.git
cd quarkus-quickstarts/hibernate-reactive-quickstart/
```
2) Add default schema (**quarkus.hibernate-orm.database.default-schema**) and enable creating schema on start (**quarkus.hibernate-orm.database.generation.create-schemas**):
```
cat <<EOF > src/main/resources/application.properties
%prod.quarkus.datasource.db-kind=postgresql
%prod.quarkus.datasource.username=quarkus_test
%prod.quarkus.datasource.password=quarkus_test

quarkus.hibernate-orm.database.generation=drop-and-create
quarkus.hibernate-orm.log.sql=true
quarkus.hibernate-orm.sql-load-script=import.sql
quarkus.hibernate-orm.database.default-schema=testschema
quarkus.hibernate-orm.database.generation.create-schemas=true

# Reactive config
%prod.quarkus.datasource.reactive.url=vertx-reactive:postgresql://localhost/quarkus_test
EOF
```
3) Add schema name to the import script
```
cat <<EOF > src/main/resources/import.sql
INSERT INTO testschema.known_fruits(id, name) VALUES (1, 'Cherry');
INSERT INTO testschema.known_fruits(id, name) VALUES (2, 'Apple');
INSERT INTO testschema.known_fruits(id, name) VALUES (3, 'Banana');
EOF
```
4) Run application, open http://localhost:8080 and look at the query in logs
```
./mvnw quarkus:dev

Hibernate:                                                                                                                                                                                                  
    select
        fruit0_.id as id1_0_,
        fruit0_.name as name2_0_
    from
        testschema.known_fruits fruit0_
    order by
        fruit0_.name
```
5) Try adding a new entry and get an exception with sequence **relation "known_fruits_id_seq" does not exist (42P01)**
```
Hibernate: 
    select
        nextval ('known_fruits_id_seq')

2023-03-13 23:21:10,314 ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (vert.x-eventloop-thread-2) HTTP Request to /fruits failed, error id: 34cb83bc-db95-4649-8977-94419376602c-1: javax.persistence.PersistenceException: org.hibernate.HibernateException: java.util.concurrent.CompletionException: io.vertx.pgclient.PgException: ERROR: relation "known_fruits_id_seq" does not exist (42P01)
```
6) Add **schema="testschema"** to the sequence for Fruit Entity https://github.com/quarkusio/quarkus-quickstarts/blob/main/hibernate-reactive-quickstart/src/main/java/org/acme/hibernate/reactive/Fruit.java#L18
```
@SequenceGenerator(name = "fruitsSequence", sequenceName = "known_fruits_id_seq", schema="testschema", allocationSize = 1, initialValue = 10)
```
7) Try to add record again, everything is OK
```
Hibernate: 
    select
        nextval ('testschema.known_fruits_id_seq')

Hibernate: 
    insert
    into
        testschema.known_fruits
        (name, id)
    values
        ($1, $2)
```

**Comment by olegsmetanin** [comment]
https://github.com/quarkusio/quarkus-quickstarts/issues/1249
https://github.com/hibernate/hibernate-reactive/issues/1120

**Comment by DavideD** [comment]
It seems to work fine to me.
Feel free to double check before closing the issue.
This is the branch with the quickstart I've updated to test the issue: https://github.com/DavideD/quarkus-quickstarts/tree/quarkus-31926

**Comment by yrodiere** [comment]
Alright, closing. Thanks for looking into this.

@olegsmetanin maybe you were using an older version of Quarkus? It definitely seems fixed in 3.1.

If you manage to reproduce this with a maintained version of Quarkus, feel free to ping us here with a reproducer.

**Comment by olegsmetanin** [comment]
The issue was observed with Quarkus 2.16.0.Final/Hibernate Reactive 5.6.14.Final (actual version as of 2023-03-14).
I confirm that the issue is fixed in Quarkus 3.0.1.Final/Hibernate Reactive 2.0.0.CR1 and Quarkus 3.1.0.Final/Hibernate Reactive 2.0.0.CR1. Thank you.
